sim/core/unit.go

```diff
@@ -425,8 +426,8 @@ func (unit *Unit) Armor() float64 {
 	return unit.PseudoStats.ArmorMultiplier * unit.stats[stats.Armor]
 }
 
-func (unit *Unit) BlockValue() float64 {
-	return unit.PseudoStats.BlockValueMultiplier * unit.stats[stats.BlockValue]
```

Comment:
If `PsuedoStats.BlockValueMultiplier` is no longer used anywhere, then that should also be removed for clarity.